data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/configuration/StreamConfig.java

```diff
@@ -12,8 +12,15 @@ public class StreamConfig {
     @JsonProperty(value = "start_position")
     private StreamStartPosition startPosition = StreamStartPosition.LATEST;
 
+    @JsonProperty("use_old_image_for_deletes")
+    private boolean useOldImageForDeletes = false;
```

Comment:
There may be room for a more extendible config here, but currently it is required that users have either `NEW_AND_OLD_IMAGE` or `NEW_IMAGE`. 

`OLD_IMAGE` specifically is needed for REMOVE actions, because otherwise an empty item is provided from the stream record.

Reply:
Maybe something like this mapping makes sense

```
stream:
   insert_image: new
   modify_image: new
   remove_image: old
```

Reply:
We can start with just

```
stream:
  image_on_remove: old // can be old or new, default to new
```

This makes the configuration more extendible in the future for something like

```
stream:
  image_on_insert: new
  image_on_modify: old_and_new
  image_on_remove: old
```